### ISSUE TITLE
NullPointerException while stopping "links localrun" with Ctrl-C

### ISSUE THREAD

**Issue by eolivelli** [apache/pulsar#9340]
**Describe the bug**

I ofter see this error while stopping a "localrun" of a sink with Ctrl-C

```
Exception in thread "main" java.lang.NullPointerException
	at org.apache.pulsar.functions.LocalRunner.start(LocalRunner.java:360)
	at org.apache.pulsar.functions.LocalRunner.main(LocalRunner.java:168)

```

**To Reproduce**
1. Run a sink with localrun and press ctrl-C

**Expected behavior**
No exception to be logged, as it may look quite scary for the user

**Comment by mattisonchao** [comment]
I can work on it. :)

**Comment by codelipenghui** [comment]
Thanks @mattisonchao, I have assigned the issue to you.

**Comment by mattisonchao** [comment]
@eolivelli  I can not reproduce this issue. 

Do you have any idea to help me? thanks :)

**Comment by eolivelli** [comment]
probably it has been fixed ?

what happens in "LocalRunner.java:360" ?

**Comment by mattisonchao** [comment]
@eolivelli  This bug has been fixed by #12278 

I think we need to close this issue.

**Comment by eolivelli** [comment]
Thank you @mattisonchao
